core/cog-view.h

```diff
@@ -32,11 +33,22 @@ struct _CogViewClass {
 
 #define COG_TYPE_VIEW_IMPL (cog_view_get_impl_type())
 
+COG_API
 GType                    cog_view_get_impl_type(void);
+
+COG_API
 CogView                 *cog_view_new(const char *first_property_name, ...);
+
+COG_API
```

Comment:
Do we need this object accessible from other part than the platforms plugins? Should not this be an internal detail hidden by cog?

Reply:
ditto.